### ISSUE TITLE
Some missing configs

### ISSUE THREAD

**Issue by The-ArdaS** [Elody-07/AWR-Adaptive-Weighting-Regression#15]
Hi there,
Thanks for sharing the training code.
I'm trying to recreate your work and it seems that some configs are missing from the config.py file (system, optimizer and scheduler). 
What configs did you use for the Hourglass model?
Thanks

**Comment by Elody-07** [comment]
Hi,

Thanks for your interest. config `system` is 'uvd', `optimizer` is 'adam' scheduler , `scheduler` is 'step' with step_size 30.

**Comment by The-ArdaS** [comment]
Thank you!

**Comment by The-ArdaS** [comment]
Hi again!
Would you please share the config for resnets too?
How much ram did resnets need?
Thanks

**Comment by Elody-07** [comment]
Hi,

We update codes and modify the config file with the params you mentioned above. Thanks for pointing it out for us! 

Since most experiments of our paper is based on ResNet, you can find most of the config in the paper.

We do have data augmentation of translation, rotation and scaling. The number of images didn't change in each epoch, but the image translate, rotate, scale or stay unchanged randomly. You can find it out by visualizing images in each epoch.
